recipes/archicad-apidevkit/all/test_package/conanfile.py

```diff
@@ -0,0 +1,12 @@
+from conan import ConanFile
+from conans import CMake
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def test(self):
```

Comment:
```suggestion
    def build(self):
```

The test method should run something, not build. As you are packaging pre-built binaries, you can run some tool packaged.

Reply:
As already told few days ago, we do not have any tools in the package. That's why I tried to put up a small buildable example but it seems that a particular tool (v142 toolset) is missing. I'm not sure why because we have some projects which are using that tool on Github without any problem. Can you tell me what to do?

Reply:
I noted in another comment that the toolset needs to be inspected in the `validate()` method. I'm looking at the CCI CI logs and it does not appear that the profiles have the `compiler.toolset` option set. Is this an oversight in CCI?

Reply:
You can check for the compiler version, see my comment above